saga-sdk-java/src/main/java/games/mythical/saga/sdk/client/SagaItemClient.java

```diff
@@ -125,10 +125,11 @@ public String issueItem(List<SagaIssueItem> issueItems,
         }
     }
 
-    public String transferItem(String inventoryId, String destOauthId) throws SagaException {
+    public String transferItem(String inventoryId, String destOauthId, Boolean prefundGas) throws SagaException {
```

Comment:
Has this been asked for?

Reply:
yeah, @lancekrogers mentioned he'd like it